packages/js/api-client/src/types.ts

```diff
@@ -0,0 +1,12 @@
+import type { Client } from "openapi-fetch"
+
+import type { paths } from "./generated/openverse"
+
+export type { components } from "./generated/openverse"
```

Comment:
Exporting components lets consumers of the package reference specific models. For example, to get the Image model (say, to define it in a store), you would use `components["schemas"].Image`, with `components` imported from `@openverse/api-client`.

Reply:
I think the paths should be exported as well, for example to allow the production of types like:

```ts
    type Method = paths["/v1/images/{identifier}/"]["get"]
    type SuccessResponse = Method["responses"][200]
    type SuccessBody = SuccessResponse["content"]["application/json"]
```

which can be quite useful to consumers who want to create type-safe client.GET arguments outside of the function call, as an example use case.